code/http4s/src/main/scala/com/avast/datadog4s/extension/http4s/MetricsOpsBuilder.scala

```diff
@@ -0,0 +1,39 @@
+package com.avast.datadog4s.extension.http4s
+
+import cats.effect.Sync
+import cats.effect.concurrent.Ref
+import com.avast.datadog4s.api.{ MetricFactory, Tag }
+import com.avast.datadog4s.extension.http4s.DatadogMetricsOps.ClassifierTags
+import com.avast.datadog4s.extension.http4s.MetricsOpsBuilder.defaultClassifierTags
+import com.avast.datadog4s.extension.http4s.impl.{ ActiveConnections, DefaultMetricsOps }
+import org.http4s.metrics.MetricsOps
+import cats.syntax.functor._
+
+private[http4s] final class MetricsOpsBuilder[F[_]: Sync](metricFactory: MetricFactory[F]) {
+  private[this] var distributionBasedTimers: Boolean = false
```

Comment:
Nicer solution would be to use custom copy method as seen for example [here](https://github.com/http4s/http4s/blob/b2892d3c61336d532c368c2490d4d0d7f91cd72b/blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala)

Reply:
fixed